neo/core/regionofinterest.py

```diff
@@ -0,0 +1,105 @@
+class RegionOfInterest:
+    pass
+
+
+class CircularRegionOfInterest(RegionOfInterest):
+
+    def __init__(self, y, x, r):
```

Comment:
The order of arguments should be consistent for `CircularRegionOfInterest` and `RectangularRegionOfInterest`. I suggest having `x, y` for both, rather than `y, x`

I think it would also be better to use `radius` rather than `r`